src/uct/base/uct_md.c

```diff
@@ -592,6 +592,10 @@ ucs_status_t uct_md_dummy_mem_reg(uct_md_h md, void *address, size_t length,
                                   const uct_md_mem_reg_params_t *params,
                                   uct_mem_h *memh_p)
 {
+    /* For testing we have to make sure that
+     * memh_h != UCT_MEM_HANDLE_NULL
```

Comment:
what is memh_h?
maybe ucs_assert(memh_p != NULL)?

Reply:
Changed to memh_p. This comment was copied from uct_cma_mem_reg.

Reply:
maybe it's not relevant anymore? can you check what happens if we remove the static assert?